manifests/resource-topology-exporter-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rte
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: resource-topology-exporter-config
```

Comment:
note that the `-config` suffix is redundant for stuff we put under `/etc`, because `/etc` is exactly the filesystem location which holds configuration

Reply:
I basically followed conventions from other projects like topology-aware-scheduler:
https://github.com/k8stopologyawareschedwg/scheduler-plugins/blob/TopologyAwareSchedulerPerNUMA/manifests/noderesourcetopology/deploy.yaml#L50